Scripts/Python/xOptionsMenu.py

```diff
@@ -1940,6 +1952,18 @@ def IShowMappedKeys(self,dlg,mapRow1,mapRow2):
                 # disable this field
                 field.hide()
 
+    def IMatchIniToGame(self):
+        km = ptKeyMap()
+        for control_code in defaultControlCodeBinds.keys():
+            if type(control_code) == type(''):
+                key1 = km.convertVKeyToChar(km.getBindingKeyConsole(control_code), km.getBindingFlagsConsole(control_code))
+                xIniInput.SetConsoleKey('"' + control_code + '"', key1 + ',')
+            else:
+                controlStr = km.convertControlCodeToString(control_code)
+                key1 = km.convertVKeyToChar(km.getBindingKey1(control_code), km.getBindingFlags1(control_code))
+                key2 = km.convertVKeyToChar(km.getBindingKey2(control_code), km.getBindingFlags2(control_code))
+                xIniInput.SetControlKey('"' + controlStr + '"', key1 + ',', key2 + ',')
```

Comment:
These string concats are very difficult for me to read because it uses two different kinds of quotation marks. I'd prefer to see these become f-strings.